src/app/router/Options/Options.tsx

```diff
@@ -28,6 +28,41 @@ import ChooseConnector from "~/app/screens/connectors/ChooseConnector";
 import ChooseConnectorPath from "~/app/screens/connectors/ChooseConnectorPath";
 import i18n from "~/i18n/i18nConfig";
 
+function renderRoutes(
+  routes:
+    | {
+        path: string;
+        element?: JSX.Element;
+        title: string;
+        logo: string;
+        children?: {
+          index?: boolean;
+          element: JSX.Element;
+          path?: string;
+        }[];
+      }[]
+    | {
+        index?: boolean;
+        element: JSX.Element;
+        path?: string;
+      }[]
+) {
+  return routes.map((route) => {
+    if ("children" in route && route.children) {
```

Comment:
Why not just `if (route?.children) {`

Reply:
The `route` can also be a parent route for type guarding.